### ISSUE TITLE
Update outdated dependencies (but only after vite bundling is merged in mirador core)

### ISSUE THREAD

**Issue by lutzhelm** [ProjectMirador/mirador-dl-plugin#71]
`npm install` lists several dependencies that are no longer maintained. Some of those dependencies are transitive dependencies via mirador, so breaking down the necessary details should be postponed until ProjectMirador/mirador#3985 is merged (which should already get rid some of those warnings).
